### ISSUE TITLE
Dataloader

### ISSUE THREAD

**Issue by joh-schb** [joh-fischer/PlantLDM#1]
Write the dataloader for `CIFAR10` and for `Plant300k`.

- [x] make data wrapper with `batch_size` etc as parameters to easily access both, `CIFAR10` and `Plant300k`
- [x] Test, Train, Val
- [x] normalize images to range $[-1, 1]$
- [x] scale up images to 255: `lambda x: x*255`
- [x] remove random crop
- [x] Create Resize and clean script
- [x] Decide which image size we want as input for our training pipeline (-> We create a 64x64 and 128x128 dataset)
    - [x] resize `Plant300k` and make folder with resized and cropped images
